### DISCUSSION TITLE
Phalcon Migrations run

### DISCUSSION THREAD

**Discussion by zivox** [phalcon/cphalcon#15400] [Q&A] [answered]
Hello all,
Posting here as I cannot find a discussion panel for Migrations. 
I've just started introducing phalcon-migrations into our database migration features. I've currently generated a handful of migrations, however they do not seem to be available to phalcon-migrations

![image](https://user-images.githubusercontent.com/36434593/114595024-8d31c580-9c53-11eb-9b7a-5783e9559067.png)

I'm assuming it is something minor but I can see all of the migrations that have been made (1.0.0,1.0.1,1.0.2,1.0.3) in the migrations/db/migrations directory.

**Comment by Jeckerson** [comment, marked as answer]
Hello

Check this configuration example:
https://github.com/phalcon/migrations#create-configuration-file
It contains `migrationsDir` where you can specify relative or absolute path of your migrations. Also, try to execute migrations from your root of project and not inside `vendor` directory.

Project structure example:
```
app/
db/migrations/
public/
migrations.php
```

And commands examples:
```
vendor/bin/phalcon-migrations generate --config=migrations.php
ls -la db/migrations <--- here will be your first generated migrations
```

**Reply by zivox** [reply]
Hey there, thank you for the reminder on root project directory. I have made the adjustments and am still receiving the same error. `migrationsDir` is set to db/migrations.

![image](https://user-images.githubusercontent.com/36434593/114606452-e18f7200-9c60-11eb-999e-59c9f4e3c97f.png)

**Reply by zivox** [reply]
AH! The problem here seemed to be `'migrationsTsBased' => true`. Thank you
